pinot-core/src/main/java/org/apache/pinot/core/query/reduce/ExecutionStatsAggregator.java

```diff
@@ -364,25 +357,6 @@ public void setStats(@Nullable String rawTableName, BrokerResponseNative brokerR
     }
   }
 
-  public void setStageLevelStats(@Nullable String rawTableName, BrokerResponseStats brokerResponseStats,
-      @Nullable BrokerMetrics brokerMetrics) {
-    if (_enableTrace) {
-      setStats(rawTableName, brokerResponseStats, brokerMetrics);
-      brokerResponseStats.setOperatorStats(_operatorStats);
-    }
-
-    brokerResponseStats.setNumBlocks(_numBlocks);
-    brokerResponseStats.setNumRows(_numRows);
-    brokerResponseStats.setMaxRowsInJoinReached(_maxRowsInJoinReached);
-    brokerResponseStats.setNumGroupsLimitReached(_numGroupsLimitReached);
-    brokerResponseStats.setStageExecutionTimeMs(_stageExecutionTimeMs);
-    brokerResponseStats.setStageExecutionUnit(_stageExecutionUnit);
-    brokerResponseStats.setTableNames(new ArrayList<>(_tableNames));
-    if (_stageExecStartTimeMs >= 0 && _stageExecEndTimeMs >= 0) {
-      brokerResponseStats.setStageExecWallTimeMs(_stageExecEndTimeMs - _stageExecStartTimeMs);
-    }
-  }
-
```

Comment:
This method is not used anymore. Instead we merge stats in `MultiStageOperator.Type.mergeInto`